guardrails/classes/history/call.py

```diff
@@ -198,8 +199,17 @@ def parsed_outputs(self) -> Stack[Union[str, Dict]]:
         return Stack(*[i.outputs.parsed_output for i in self.iterations])
 
     @property
+    @deprecated(
+        """'Call.validation_output' is deprecated and will be removed in \
```

Comment:
The message has to be a literal or pylance won't display the property as deprecated (i.e. with a strikethrough and deprecation message)